panoptes_aggregation/extractors/pluck_extractor.py

```diff
@@ -0,0 +1,35 @@
+from slugify import slugify
+from .extractor_wrapper import extractor_wrapper
+from .utilities import pluck_fields
+
+
+def slugify_or_null(s):
+    '''Slugify value while casting `null` as a string fisrt'''
+    if (s is None) or (isinstance(s, bool)) or (isinstance(s, int)):
+        return str(s)
+    else:
+        return slugify(s, separator='-')
```

Comment:
Import this function from the question extractor rather than rewrite it:
```python
from .question_extractor import slugify_or_null
```

Reply:
I removed the pluck_extractor completely since it's functionality is now tied to the other extractors.